examples/nextjs/starter/codegen.ts

```diff
@@ -7,12 +7,17 @@ import 'dotenv/config';
 const GRAPHQL_GLOB = './src/**/*.graphql';
 const graphqlFiles = globSync( GRAPHQL_GLOB );
 
+// If there's no explicit SITE_URL, it's the same as the HOME_URL.
+const homeUrl =
+	process.env.WP_SITE_URL ||
+	process.env.NEXT_PUBLIC_WP_SITE_URL ||
+	process.env.NEXT_PUBLIC_WP_HOME_URL;
+
+const graphqlEndpoint =
+	process.env.GRAPHQL_ENDPOINT || process.env.NEXT_PUBLIC_GRAPHQL_ENDPOINT;
+
```

Comment:
This is a userland example file, we don't need to fallback here. If they change their .env file vars, they should update this themselves.